types/index.d.ts

```diff
@@ -289,10 +289,61 @@ declare namespace PhonegapPluginPush {
 		from?: string
 		notId?: string
 	}
-
+  /**
+   * Set Channel Variable,once created cannot be changed programmtically for importance and vibration
+   */
+  interface channel{
```

Comment:
This should be upper case, as it's a proper type name.
```suggestion
  interface Channel {
```